tests/DB/QueryBuilderTest.php

```diff
@@ -3,28 +3,72 @@
 namespace Message\Cog\Test\DB;
 
 use Message\Cog\DB\QueryBuilder;
-use Mockery as m;
 
-class QueryBuilderTest extends \PHPUnit_Framework_TestCase
+class QueryBuilderTestRefactor extends \PHPUnit_Framework_TestCase
 {
 	private $_builder;
 	private $_parser;
 	private $_connect;
 
 	public function setUp()
 	{
-		$this->_connect = m::mock('Message\Cog\DB\Adapter\ConnectionInterface');
-		$this->_parser  = m::mock('Message\Cog\DB\QueryParser');
+		$this->_connect = $this->getMockBuilder('Message\\Cog\\DB\\Adapter\\ConnectionInterface')->disableOriginalConstructor()->getMock();
+		$this->_parser  = $this->getMockBuilder('Message\\Cog\\DB\\QueryParser')->setConstructorArgs(array($this->_connect))->setMethods(array('escape'))->getMock();
 		$this->_builder = new QueryBuilder($this->_connect, $this->_parser);
 	}
-	
+
+	public function testRun()
+	{
+		$result = $this->getMockBuilder('Message\\Cog\\DB\\Adapter\\ResultInterface')
+			->setMethods(array('fetchArray', 'fetchObject', 'seek', 'getAffectedRows', 'getLastInsertId'))
+			->setConstructorArgs(array(null, $this->_connect))
+			->getMockForAbstractClass()
+									;
+
+		$this->_connect->expects($this->once())
+			->method('query')
+			->will($this->returnValue($result));
+
+		$query = $this->_builder
+			->select("*")
+			->from('table')
+			->getQuery()
+			->run();
+
+	}
+
+	public function testAddParams()
+	{
+		$parser  = $this->getMockBuilder('Message\\Cog\\DB\\QueryParser')->setConstructorArgs(array($this->_connect))->setMethods(array('parse'))->getMock();
+		$builder = new QueryBuilder($this->_connect, $parser);
+
+		// Configure the stub.
+		$parser->method('parse')
+					 ->willReturn(true)
+					 ;
+
+		// Test that getQueryString is calling parse
+		$parser->expects($this->once())
+			->method('parse')
+			->with(
+						"SELECT\n*\nFROM table",
```

Comment:
Please can you review the formatting here, so things are inline when they indent and don't indent too far (should be one tab per level)